src/sagemaker/jumpstart/model.py

```diff
@@ -312,6 +314,27 @@ def _is_valid_model_id_hook():
 
         super(JumpStartModel, self).__init__(**model_init_kwargs.to_kwargs_dict())
 
+    def retrieve_default_payload(self) -> JumpStartSerializablePayload:
+        """Returns default payload associated with the model.
+
+        Payload can be directly used with the `sagemaker.predictor.Predictor.predict(...)` function.
+        """
+        sample_payloads: Optional[List[JumpStartSerializablePayload]] = payloads.retrieve_samples(
+            model_id=self.model_id,
+            model_version=self.model_version,
+            region=self.region,
+            tolerate_deprecated_model=self.tolerate_deprecated_model,
+            tolerate_vulnerable_model=self.tolerate_vulnerable_model,
+            sagemaker_session=self.sagemaker_session,
+        )
+
+        if sample_payloads is None or len(sample_payloads) == 0:
+            raise NotImplementedError(
+                f"No default payload supported for model ID '{self.model_id}'."
```

Comment:
nit: error vs returning None?

Reply:
Not sure, i'm cool with both

Reply:
are you now? That's something you should definitely have an opinion on Evan.

Reply:
What I mean is that both experiences can be justified from a UX perspective.